lib/strava/models/mixins/start_date_local.rb

```diff
@@ -0,0 +1,27 @@
+module Strava
+  module Models
+    module Mixins
+      module StartDateLocal
+        extend ActiveSupport::Concern
+
+        included do
+          property 'start_date_local', transform_with: ->(v) { ::Time.parse(v) }
+          def start_date_local
+            shift = begin
+                if timezone.include?('+')
+                  "+0#{timezone.split('+').last.to_i}:00"
+                elsif timezone.include?('-')
+                  "-0#{timezone.split('-').last.to_i}:00"
+                end
+                    rescue NameError
```

Comment:
When would this happen? 

We shouldn't rely on an exception for a parser.

Reply:
in a lap, no timezone is given

Reply:
Don't rely on an exception, should be an `if/unless`

Reply:
implemented a guard clause using #try